suguar.go

```diff
@@ -0,0 +1,104 @@
+package vshard_router //nolint:revive
+
```

Comment:
Typo in filename `suguar.go` -> `sugar.go`?

Reply:
done